kai_mcp_solution_server/tests/mcp_client.py

```diff
@@ -573,7 +593,7 @@ def main() -> None:
     parser.add_argument(
         "--insecure",
         action="store_true",
-        help="Allow insecure connections (skip SSL verification)",
+        help="Allow insecure connections (skip SSL verification for http transport",
```

Comment:
Nit: Add closing parenthesis

```suggestion
        help="Allow insecure connections (skip SSL verification for http transport)",
```